migration/src/refactor_platform_rs.sql

```diff
@@ -96,6 +98,8 @@ COMMENT ON COLUMN "refactor_platform"."organizations"."name" IS 'The name of the
 
 COMMENT ON COLUMN "refactor_platform"."organizations"."logo" IS 'A URI pointing to the organization''s logo icon file';
 
+COMMENT ON COLUMN "refactor_platform"."organizations"."slug" IS 'Canonical name for the record. Considered immutable by convention';
```

Comment:
```suggestion
COMMENT ON COLUMN "refactor_platform"."organizations"."slug" IS 'A human-friendly canonical name for a record. Considered immutable by convention. Must be unique.';
```

Reply:
updated in [b4f0003](https://github.com/refactor-group/refactor-platform-rs/pull/95/commits/b4f000334109d936bf766c8da3ea8c42ba9b79c1)